tests/sentry/monitors/test_tasks.py

```diff
@@ -109,6 +113,49 @@ def test_missing_checkin(self, mark_environment_missing_mock):
         ).replace(second=0, microsecond=0)
         assert missed_checkin.monitor_config == monitor.config
 
+    @mock.patch("sentry.monitors.tasks.mark_environment_missing")
+    def test_temp_ingore_next_checkin_equal_latest(self, mark_environment_missing_mock):
+        org = self.create_organization()
+        project = self.create_project(organization=org)
+
+        task_run_ts, ts = make_ref_time()
+
+        monitor = Monitor.objects.create(
+            organization_id=org.id,
+            project_id=project.id,
+            type=MonitorType.CRON_JOB,
+            config={
+                "schedule": "* * * * *",
+                "schedule_type": ScheduleType.CRONTAB,
+                "max_runtime": None,
+                "checkin_margin": None,
+            },
+        )
+
```

Comment:
can we add a test for a monitor that does have a `checkin_margin` set and confirm that it's still included in the mock calls? that can isolate the `Q(next_checkin=F("next_checkin_latest"))` logic

Reply:
or is that covered in later tests?

Reply:
Nah it's not covered, good idea 👍

Reply:
Oh, no you're right, the other test cases do cover it 👍